Pipfile

```diff
@@ -14,7 +14,7 @@ pylint-quotes = "*"
 "beautifulsoup4" = "*"
 httmock = "*"
 "flake8" = "*"
-"flake8-blind-except" = "*"
+flake8-blind-except = "==v0.1.1"
```

Comment:
From https://github.com/elijahandrews/flake8-blind-except/pull/12 and https://github.com/elijahandrews/flake8-blind-except/issues/11 - it looks like the functionality of this version is now included in flake8 as E722. We should consider dropping this dependency rather than fix on this version.

Reply:
Done 👍🏻